ironfish/src/rpc/routes/wallet/postTransaction.ts

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import * as yup from 'yup'
+import { RawTransactionSerde } from '../../../primitives/rawTransaction'
+import { ApiNamespace, router } from '../router'
+
+export type PostTransactionRequest = { transaction: string }
+
+export type PostTransactionResponse = {
+  transaction: string
+}
+
+export const PostTransactionRequestSchema: yup.ObjectSchema<PostTransactionRequest> = yup
+  .object({
+    transaction: yup.string().defined(),
+  })
+  .defined()
+
+export const PostTransactionResponseSchema: yup.ObjectSchema<PostTransactionResponse> = yup
+  .object({
+    transaction: yup.string().defined(),
+  })
+  .defined()
+
+router.register<typeof PostTransactionRequestSchema, PostTransactionResponse>(
```

Comment:
This is missing on the client, we should add a function on the RpcClient as well like all the other rpcs.